projects/js-packages/components/components/contextual-upgrade-trigger/index.tsx

```diff
@@ -1,18 +1,30 @@
 import { Icon, arrowRight } from '@wordpress/icons';
+import classnames from 'classnames';
 import Text from '../text';
 import styles from './style.module.scss';
 import { CutBaseProps } from './types';
 import type React from 'react';
 
-const ContextualUpgradeTrigger: React.FC< CutBaseProps > = ( { description, cta, onClick } ) => {
+const ContextualUpgradeTrigger: React.FC< CutBaseProps > = ( {
+	description,
+	cta,
+	onClick,
+	href,
+	className,
+} ) => {
+	const Tag = href !== undefined ? 'a' : 'button';
+	const tagProps = Tag === 'a' ? { href, target: '_blank' } : { onClick };
```

Comment:
It might be overkill, but this seems to replicate the internals of the `@wordpress/components` Button. That package is a dependency, so would it be worth using it here?

Reply:
That seems like something to address. If the button component can't be used, and we end up implementing a new version for specific situations then that means the component is less useful.

Although I don't think it's something we should change with this PR.

Reply:
> That seems like something to address

I don't think so, we have some design specifications for the core button and jetpack button, ofc we don't want to move out of it all the time, but sometimes (if it makes sense) don't think it is a huge problem. It is really difficult to have a component fitting all scenarios, for designers too,  and in this case, we use the pure version of it.

What we try to avoid in the components is overriding a piece of the component, this for sure is a break in our specifications, and needs to be avoided.